README.md

```diff
@@ -35,34 +37,53 @@ The required tables can be created executing:
+
+The poll interval should be short enough to process messages in time, and long
+enough to minimize resource consumption for the database. Boost mode is a 
+solution which automatically handles spikes by processing messages in quick 
+succession. When no new messages appear for a specified time, boost mode is
+disabled again.
+
+The frequency by which the queue loop will look for new messages is the 
+configured `pollInterval`. In boost mode, the option `boostPollInterval` is 
+used instead. `boostTime` defines the time since the last processed message 
+after which boost mode is deactivated again.
+
 ## Specific options
 
 The `DoctrineQueue` supports following options:
 
-| Option                  | Type    | Default                                 | Description                              |
-| ----------------------- |---------| ---------------------------------------:| ---------------------------------------- |
-| defaultTimeout          | integer | 60                                      | Number of seconds new messages are waited for before a timeout occurs (This is overridden by a "timeout" argument in the `waitAndTake()` and `waitAndReserve()` methods |
-| pollInterval            | integer | 1                                       | Number of seconds between SQL lookups for new messages |
-| tableName               | string  | flowpack_jobqueue_messages_<queue-name> | Name of the database table for this queue. By default this is the queue name prefixed with "flowpack_jobqueue_messages_" |
-| backendOptions          | array   | -                                       | Doctrine-specific connection params (see [Doctrine reference](http://doctrine-orm.readthedocs.io/projects/doctrine-dbal/en/latest/reference/configuration.html))|
+| Option            | Type    |                                 Default | Description                                                                                                                                                             |
+|-------------------|---------|----------------------------------------:|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| defaultTimeout    | integer |                                      60 | Number of seconds new messages are waited for before a timeout occurs, this is overridden by a "timeout" argument in the `waitAndTake()` and `waitAndReserve()` methods |
+| pollInterval      | float   |                                       1 | Number of seconds between SQL lookups for new messages                                                                                                                  |
+| boostPollInterval | float   |                                     0.5 | Number of seconds between SQL lookups for new messages when in "boost mode"                                                                                             |
```

Comment:
The float is a compromise to make the change non breaking but consistent I assume? Because I would spontaneously prefer integer microseconds, although this format is nice.

Reply:
That, and also seconds make it less likely "insanely low" intervals are given. People probably think a bit more when giving 0.0002 instead of 200. ;)